modules/core/src/main/java/org/apache/ignite/spi/discovery/tcp/ServerImpl.java

```diff
@@ -1214,6 +1231,46 @@ else if (spiState == LOOPBACK_PROBLEM) {
             .end();
     }
 
+    /**
+     * Schedules a maintenance task to clear cache folders based on the exception message.
+     *
+     * @param chackFailedMsg Exception message.
+     */
+    private void scheduleMaintenanceTaskToClearCacheFolders(TcpDiscoveryCheckFailedMessage chackFailedMsg) {
+        ArrayList<CacheConfiguration> cacheCfgs = new ArrayList<>();
+
+        for (String cacheName : chackFailedMsg.error()
+            .replaceFirst("^.*\\[", "")
+            .replaceFirst("\\].*", "")
+            .split(", ")) {
+            CacheConfiguration cc = gridKernalContext().cache().context().cacheContext(CU.cacheId(cacheName)).config();
+
+            cacheCfgs.add(cc);
+        }
+
+        FilePageStoreManager pageStore = (FilePageStoreManager) gridKernalContext().cache().context().pageStore();
+
+        try {
+            String params = cacheCfgs.stream()
+                .map(ccfg -> pageStore.cacheWorkDir(ccfg).getName())
+                .collect(Collectors.joining(File.separator));
+
+            log.info("Task params: " + params + " sep: " + File.separator);
```

Comment:
This log message is of a very little value for end user and/or CS engineer. Please rewrite this message to contain useful information: what happened (old caches were found, list of caches specified), what is going to happen (node is scheduled to enter Maintenance Mode to automatically clean old caches).